surveysite/settings.py

```diff
@@ -147,6 +148,7 @@
     'survey',
     'debug_toolbar',
```

Comment:
Given that we don't need this in production, I think you can probably abstract this out into the settings_local.py as well.

Keep the option open so that we can do database read/write optimizations in the future.